portal-kernel/src/com/liferay/portal/kernel/workflow/WorkflowConstants.java

```diff
@@ -107,6 +109,8 @@
 
 	public static final int TYPE_COMPLETE = 10001;
 
+	public static final String VIEW_TASK_NOTIFICATION = "VIEW_NOTIFICATION";
```

Comment:
Hi @seltong,

Please, rename the value of this variable to VIEW_TASK_NOTIFICATION

Thanks!